core/modules/user/user.module

```diff
@@ -789,7 +789,14 @@ function user_account_form(&$form, &$form_state) {
     '#maxlength' => USERNAME_MAX_LENGTH,
     '#description' => t('Spaces are allowed; punctuation is not allowed except for periods, hyphens, apostrophes, and underscores.'),
     '#required' => TRUE,
-    '#attributes' => array('class' => array('username')),
+    '#attributes' => array(
+      'class' => array('username'),
+      // Add attributes to field to prevent spell-jacking and unwanted
+      // automatic changes caused by the browser.
+      'autocapitalize' => 'none',
+      'autocorrect' => 'off',
+      'spellcheck' => 'false',
+    ),
```

Comment:
This block simply adds the desired attributes to both name and password for this specific high-level form.